ui_framework/components/_variables.scss

```diff
@@ -80,3 +80,6 @@ $toolBarItsemSpacing: 10px;
 $globalFormControlHorizontalPadding: 12px;
 $globalFormControlPadding: 3px $globalFormControlHorizontalPadding 4px;
 $globalFormInputHeight: 30px;
+$globalFormFieldDefaultWidth: 180px;
+$globalFormFieldSmallWidth: 60px;
+$globalFormFieldLargeWidth: 400px;
```

Comment:
I like the idea of applying some rhythms that make sense, but as it is they're just chosen because they look good in different scenarios. Can we get together at some point and figure out a rhythm that we can apply here so the values aren't so arbitrary?